README.md

```diff
@@ -74,15 +81,22 @@ Finally, [eoapi-template](https://github.com/developmentseed/eoapi-template) is
 
 ## Contribution & Development
 
-See [CONTRIBUTING.md](https://github.com/developmentseed/eoAPI/blob/main/CONTRIBUTING.md)
+We highly value and rely on our community! Whether you're an expert or just getting started, you can make a difference. Here's how:
+
+- **Contribute**: Check out our [CONTRIBUTING.md](https://github.com/developmentseed/eoAPI/blob/main/CONTRIBUTING.md) guide to understand how you can contribute.
+- **Engage in Discussions**: Share your ideas, ask questions, or provide feedback through [GitHub Discussions](https://github.com/developmentseed/eoAPI/discussions). This is where most of our project conversations take place.
+- **Report Issues**: Found a bug or have a feature request? Raise it on our [issues page](https://github.com/developmentseed/eoAPI/issues).
 
+---
 ## License
 
-See [LICENSE](https://github.com/developmentseed/eoAPI/blob/main/LICENSE)
+At Development Seed, we believe in open collaboration and making tools and data more accessible. In line with this ethos, we've explicitly chosen a less restrictive license for `eoAPI`.
+
+For full license details, see [LICENSE](https://github.com/developmentseed/eoAPI/blob/main/LICENSE).
 
 ## Authors
 
-Created by [Development Seed](<http://developmentseed.org>)
+Nurtured by [Development Seed](<http://developmentseed.org>)
 
 See [contributors](https://github.com/developmentseed/eoAPI/graphs/contributors) for a listing of individual contributors.
```

Comment:
This usually goes to a Contributors file, since there are many ways to contribute without writing code. Also github APIs are not a stable list that can always live in the code.
How do we plan to thank MS, NASA, AWS, etc...

Reply:
So, we opted in the past to keep recognition of funding in the specific repositories. I'll pin this for future discussion again if we don't resolve it here, but I won't let it block the PR from being merged. We can always open a new one if needed.

Reply:
☝️ the issue mentioning funders is unaddressed, will keep it open until Friday. If we can't reach a decision, I'll merge and plan a follow-up. @wildintellect @vincentsarago @bitner

Reply:
the creator of eoAPI is development seed. If we start mentioning/thanking every project/people being used in eoAPI we are not done 😅 

> How do we plan to thank MS, NASA, AWS, etc...

Well, if we get direct founding for eoAPI core development we can add a `sponsors` or `acknowledgment` section.

Reply:
You have a valid point, the funding has mostly been on the sub-projects, so that should be where the credit/acknowledgement is given.